src/cli/README_COMMISSIONER.md

```diff
@@ -18,6 +18,7 @@ See [README_COMMISSIONING.md](README_COMMISSIONING.md).
 - [sessionid](#sessionid)
 - [start](#start)
 - [stop](#stop)
+- [state](#state)
```

Comment:
nit: move up one to maintain alphabetic ordering.

Reply:
reordered